src/components/sequencing/form/EnumEditor.svelte

```diff
@@ -31,7 +30,7 @@
   function onSelectReferenceModel(event: CustomEvent<SelectedDropdownOptionValue>) {
     const { detail: enumVal } = event;
     if (typeof enumVal === 'string') {
-      setInEditor(quoteEscape(enumVal));
+      setInEditor(enumVal);
```

Comment:
Do we no longer need to worry about quotes in enums?

Reply:
We don't allow quotes on symbolic arguments. Quotes in literals should be handled by ArgEditor component

https://github.com/NASA-AMMOS/aerie-ui/blob/8a2d51256820002a945d444a5ffb4c673f0ba634/src/components/sequencing/form/ArgEditor.svelte#L114

I'll test and confirm.

Reply:
Testing looks good to me, I'll file a bug on the linter, but it's low priority as we haven't encountered quotes in enum values in any flight dictionary. We support them in free form string arguments, and we've chosen to represent both as quoted strings.

https://github.com/user-attachments/assets/1c9cee48-860b-4d88-9731-d92ba6f040a1